src/updaterDB.js

```diff
@@ -1,861 +1,12 @@
-    rollNumber: "2020BCS-061",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  suyashvsingh: {
-    name: "Suyash Vikram Singh",
-    rollNumber: "2020IMT-104",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  Manoramsharma: {
-    name: "Manoram Sharma",
-    rollNumber: "2020IMG-037",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  AbhishekSharma75: {
-    name: "Abhishek Sharma",
-    rollNumber: "2020IMT-003",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  shahlaukik: {
-    name: "Laukik Shah",
-    rollNumber: "2020BCS-050",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  shikhar13012001: {
-    name: "Shikhar Gupta",
-    rollNumber: "2020IMT-090",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  "Ayush-019": {
-    name: "Ayush Garg",
-    rollNumber: "2020IMT-019",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  saichidvi: {
-    name: "Charagondla Saichidvi",
-    rollNumber: "2020IMT-023",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  sadityakumar9211: {
-    name: "Aditya Kumar Singh",
-    rollNumber: "2020BCS-004",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  shanmai20: {
-    name: "Thota Shanmai",
-    rollNumber: "2020BCS072",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  hshrivastav: {
-    name: "Himanshu Shrivastava",
-    rollNumber: "2020IMG-028",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  impriyanshuin: {
-    name: "Priyanshu",
-    rollNumber: "2020IMT-076",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  PkSinha7: {
-    name: "Prakhar Kumar Sinha",
-    rollNumber: "2020IMT-070",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  "Syd-B": {
-    name: "Siddharth Butoliya",
-    rollNumber: "2020IMG-061",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  SatyaS18: {
-    name: "Satyakant Sahu",
-    rollNumber: "2020IMT-088",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  mayanksingh038: {
-    name: "Mayank Singh",
-    rollNumber: "2020IMG-038",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  ChoutapallyAkshara: {
-    name: "Choutapally Akshara",
-    rollNumber: "2020IMT-024",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  Firestorm2602: {
-    name: "Siddhant Salve",
-    rollNumber: "2020IMT-099",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  shambhu1212: {
-    name: "Shambhavi Singh",
-    rollNumber: "2020IMG-058",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-  Hachimitsu25: {
-    name: "Devesu Sundram Dev",
-    rollNumber: "2020IMT-027",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-};
+import projects from "../public/projects";
+import students from "../public/studentScoreCard";
```

Comment:
```suggestion
import projects from "../public/projects.js";
import students from "../public/studentScoreCard.js";
```
Add file extension, as without it a module not found error is being thrown

Reply:
Since both the file extensions are same, we need not specify `.js`

Reply:
@sohan2410 You do need to add the .js file extension. We have set our type to be a module, which means the extension is required. Take a look at [this StackOverflow answer](https://stackoverflow.com/a/67105989).

Reply:
Okay, I've made changes.